adapter/smi.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Layer5, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
```

Comment:
@mgfeller I forget where you asked about copyright previously, but did want to answer your question - all of Layer5's open projects are licensed under Apache v2. It is helpful to have the (brief) standard license header at the top of each file (conveniently done for Golang files), but not strictly required as all files within the repos are implicitly licensed under the same license. At least for now, this suffices and is common practice. We'll see if this practice doesn't change over time.

Up to the point that we don't certain projects to the CNCF, the copyright holder is Layer5, Inc. After the point of donation, we'll change that holder to Meshery Authors and SMP Authors.

Reply:
@leecalcote thanks for feedback and clarification!

Reply:
Oops. Instead of “don’t” one of the sentences above should say “donate to the CNCF”.